### PR TITLE
Watchdog

### PR DESCRIPTION
**Pull request by sdalu**
Implemented watchdog for NRF51

### CODE REVIEW

File: os/hal/ports/NRF51/NRF51822/wdg_lld.h [hunk 1]
```diff
@@ -0,0 +1,127 @@
+/*
+    ChibiOS - Copyright (C) 2006 Stephane D'Alu
```
Comment (utzig) [inline os/hal/ports/NRF51/NRF51822/wdg_lld.h]: 2016

File: testhal/NRF51/NRF51822/WDG/chconf.h [hunk 2]
```diff
@@ -0,0 +1,499 @@
+/*
+    Copyright (C) 2015 Stephen Caudle
```
Comment (utzig) [inline testhal/NRF51/NRF51822/WDG/chconf.h]: Please update the testhal files using only your name and current year.

File: os/hal/ports/NRF51/NRF51822/wdg_lld.c [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+ */
+void wdg_lld_init(void) {
+  WDGD1.state = WDG_STOP;
+  WDGD1.wdt   = NRF_WDT;
+}
+
+/**
+ * @brief   Configures and activates the WDG peripheral.
+ *
+ * @note    Once started there is now way out.
```
Comment (utzig) [inline os/hal/ports/NRF51/NRF51822/wdg_lld.c]: typo: there is *no* way out...

File: os/hal/ports/NRF51/NRF51822/wdg_lld.c [hunk 4]
```diff
@@ -0,0 +1,150 @@
+/*
+    ChibiOS - Copyright (C) 2006 Stephane D'Alu
```
Comment (utzig) [inline os/hal/ports/NRF51/NRF51822/wdg_lld.c]: 2016?

**Comment by utzig** [comment]
Looks good. Please update the issues before merge.

File: os/hal/ports/NRF51/NRF51822/wdg_lld.h (hunk 1 above)
Comment (sdalu) [inline os/hal/ports/NRF51/NRF51822/wdg_lld.h]: Time pass to fast :)

Comment (utzig) [inline os/hal/ports/NRF51/NRF51822/wdg_lld.h]: I sure know about it! :P

**Comment by sdalu** [comment]
That's should be fixed

**Comment by utzig** [comment]
Please update **all** testhal license headers!

**Pull request merged**